src/test/kotlin/org/malachite/estella/BaseIntegration.kt

```diff
@@ -251,4 +256,19 @@ class BaseIntegration {
         return Timestamp.valueOf(localDateTime)
     }
 
+
+    companion object {
+        class PropertyOverride : ApplicationContextInitializer<ConfigurableApplicationContext> {
+            override fun initialize(applicationContext: ConfigurableApplicationContext) {
+                TestPropertySourceUtils.addInlinedPropertiesToEnvironment(
+                    applicationContext,
+                    "mail_service_url=http://localhost:9797",
+                    "admin_api_key=API_KEY",
+                    "should_fake_load=false",
+                    "cloud_amqp_url=amqp://localhost:5672"
+                )
+            }
+        }
+    }
```

Comment:
To jest springowa magia która załaduje te rzeczy do kontekstu aplikacji żeby działało?